rulesets/README.md

```diff
@@ -0,0 +1,49 @@
+# Rulesets for opensafely-actions repositories
+
+This directory houses two JSON files that can be used to import rulesets into
+a repository in the `opensafely-actions` organization.
+It is recommended to apply these rulesets to repositories housing reusable actions
+within the organization.
+
+### Importing a ruleset into a repository
```

Comment:
Thank you for considering the user by writing instructions for them! That said, perhaps we could simply link to the "[Importing a ruleset][1]" section of the GitHub Docs? If we did, then we'd be considering the user _and_ saving ourselves from having to maintain these instructions.

[1]: https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-rulesets/managing-rulesets-for-a-repository#importing-a-ruleset